packages/flutter/test/animation/animation_controller_test.dart

```diff
@@ -580,6 +580,54 @@ void main() {
     statusLog.clear();
   });
 
+  test('calling repeat with reverse set to true makes the animation alternate '
```

Comment:
Should add tests for the new `repeat` method parameter asserts. Similar to the test called 'AnimationController repeat() throws if period is not specified' (and etc).

Reply:
Done 👍